src/app/models/evt-models.ts

```diff
@@ -331,3 +331,163 @@ export class Word extends GenericElement {
+}
+
+export class AltIdentifier extends GenericElement {
+    noteEl?: Note[];
+    idno?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when idno is handled
+    collection?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when collection is handled
+    repository?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when repository is handled
+    region?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when region is handled
+    settlement?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when settlement is handled
+}
+
+export class MsItemStruct extends GenericElement {
+    n?: string;
+    class?: string;
+    defective?: boolean;
+    locus?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when locus is handled
+    author?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when author is handled
+    title?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when title is handled
+    rubric?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when rubric is handled
+    finalRubric?: Array<ParseResult<GenericElement>>; // TODO: Add specific type finalRubric defective is handled
+    bibl?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when bibl is handled
+}
+
+export class MsItem extends GenericElement {
+    n?: string;
+    class?: string;
+    defective?: boolean;
+    author?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when author is handled
+    title?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when title is handled
+    rubric?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when rubric is handled
+    textLang?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when textLang is handled
+    docAuthor?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when docAuthor is handled
+    docTitle?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when docTitle is handled
+    docImprint?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when docImprint is handled
+    docDate?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when docDate is handled
+    noteEl?: Note[];
+    locus?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when locus is handled
+    locusGrp?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when locusGrp is handled
+    bibl?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when bibl is handled
+    incipit?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when incipit is handled
+    explicit?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when explicit is handled
+    filiation?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when filiation is handled
+    gapEl?: Gap[];
+}
```

Comment:
According to the TEI guidelines there are more elements that can be used to descrive the `msItem` (see https://www.tei-c.org/release/doc/tei-p5-doc/en/html/MS.html#mscoit - 10.6.1 The <msItem> and <msItemStruct> Elements); at least add those indicated in the Manuscript Description section. Moreover you should consider also the fact that it may contain only paragraphs and no other structured element

> While neither msItem nor msItemStruct may contain untagged running text, both permit an unstructured description to be provided in the form of one or more paragraphs of text. They differ in this respect also: if paragraphs are supplied as the content of an msItem, then none of the other component elements listed above is permitted; in the msItemStruct case, however, paragraphs may appear anywhere as an alternative to any of the component elements listed above.